File: 数组/数组-简单/1725. 可以形成最大正方形的矩形数目/1725. 可以形成最大正方形的矩形数目.go
func countGoodRectangles(rectangles [][]int) (ans int) {
	max := 0
	for _,v := range rectangles{
		cur := min(v[0],v[1])
		if cur > max {
			max = cur
			ans = 1
		} else if cur == max {
			ans ++
		}
	}
	return
}

func min(a,b int) int{
	if a>b {
		return b
	}
	return a
}

File: 数组/数组-简单/1725. 可以形成最大正方形的矩形数目/1725. 可以形成最大正方形的矩形数目.js
/**
 * @param {number[][]} rectangles
 * @return {number}
 */
 var countGoodRectangles = function(rectangles) {
    let max = 0, ans=0
    for (const r of rectangles) {
        const cur = Math.min(r[0],r[1])
        if (cur > max) {
            max = cur
            ans = 1
        } else if (cur==max){
            ans ++
        }
    }
    return ans
};